pkg/domain/infra/abi/network.go

```diff
@@ -49,8 +38,14 @@ func (ic *ContainerEngine) NetworkInspect(ctx context.Context, namesOrIds []stri
 	var (
 		rawCNINetworks []entities.NetworkInspectReport
 	)
+
+	config, err := ctx.Value("runtime").(*libpod.Runtime).GetConfig()
```

Comment:
can this be replaced with `ic.Libpod.GetConfig()`?

Reply:
Probably, yes. I've just seen that ctx.Value("runtime").(*libpod.Runtime) around there (e.g. for network creation) and I thought it was the way to do there.

Reply:
I think that is mostly used for remote functionality, not used under pkg/domain/infra/abi

Reply:
Okay, I update this.